application/modules/character/views/character.tpl

```diff
@@ -15,7 +15,7 @@
 	
 	<section id="armory_name">
 		<h1>{$name} <a href="{$url}guild/{$realmId}/{$guild}">{$guildName}</a></h1>
-		<h2><b>{$level}</b> {$raceName} {$className}, <i>{$realmName}</i></h2>
+		<h2 class="color-c{$class}"><b>{$level}</b> {$raceName} {$className}, <i>{$realmName}</i></h2>
```

Comment:
Would suggest

`<h2 class="color-c{$class}"><b>{$level}</b> {$raceName} {$className}</h2> <i>{$realmName}</i>`

instead as we dont want the class color to apply to the realmname as well.

Reply:
And the RealmName are usually something that **can** appear on a new line and by itself.

Reply:
how you wish , idk you can edit how you think , but the realm on the next line is kinda ugly